app/src/main/java/org/fossasia/phimpme/utilities/Utils.java

```diff
@@ -167,7 +169,7 @@ public static boolean checkNetwork(Context context, @NonNull View view) {
         if (isInternetOn(context)) {
             return true;
         } else
-            SnackBarHandler.show(view,context.getString(R.string.not_connected));
+            Toast.makeText(context,R.string.internet_is_off,Toast.LENGTH_SHORT).show();
```

Comment:
Why was this SnackBar changed to a toast?

Reply:
because this snackbar was not showing when checknetwork returned false.